src/components/Main.jsx

```diff
@@ -20,10 +20,10 @@ const getCustomersFromLocalStorage = () => {
 };
 
 const [customersData,setCustomersData] = useState(() => getCustomersFromLocalStorage());
```

Comment:
Why did you use `useState` initial function, I'm curious

Reply:
Just to optimize performance I use lazy initialization to get data from local storage only in first rendering

Reply:
Good